bee-ledger/src/workers/event.rs

```diff
@@ -81,3 +83,14 @@ pub struct PrunedIndex {
 /// An event that indicates that a receipt was created.
 #[derive(Clone)]
 pub struct ReceiptCreated(pub Receipt);
+
+/// An event that indicates that the ledger was updated.
+#[derive(Clone)]
+pub struct LedgerUpdated {
+    /// Index of the confirmed milestone.
+    pub milestone_index: MilestoneIndex,
+    /// The outputs created within the confirmed milestone.
+    pub created_outputs: HashMap<OutputId, CreatedOutput>,
+    /// The outputs consumed within the confirmed milestone.
```

Comment:
Not clear to me why there is `CreatedOutput` then

Reply:
Yeah i get this can be a bit confusing. A `ConsumedOutput` is just a record of the consumption, it doesn't actually contain an output so we associate a `CreatedOutput` to it. I'll think about changing the names later.